data/mods/ssb/statuses.ts

```diff
@@ -57,6 +57,22 @@ export const BattleStatuses: {[k: string]: ModdedPureEffectData} = {
 			}
 		},
 	},
+
+	frostyicelad: {
+		noCopy: true,
+		onStart(source) {
+			this.add(`c|${getName('Frostyicelad')}|Oh i guess its my turn now! Time to sweep!`);
+			if (source.species.id !== 'absolmega' || source.illusion) return;
```

Comment:
absol

Reply:
oops ever forget to edit the c/p when trying to be lazy?